fulp_modules/devils/devil.dm

```diff
@@ -54,7 +54,7 @@
 /mob/living/carbon/true_devil/create_internal_organs()
 	internal_organs += new /obj/item/organ/brain
 	internal_organs += new /obj/item/organ/tongue
-	internal_organs += new /obj/item/organ/eyes
+	internal_organs += new /obj/item/organ/eyes/robotic/shield
```

Comment:
bruuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuh

you can emp these (and/or extract them and be confused as to why all devils are part-robot?). just make an invincible eye subtype like you did with the ears.